onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -1197,7 +1197,7 @@ def _where_input_wrangler(
     TorchLibOpInfo("sinh", core_ops.aten_sinh),
     TorchLibOpInfo(
         "softmax",
-        special_ops.aten_special_softmax,
+        core_ops.aten_softmax,
```

Comment:
Should we add aten__softmax into test as well?

Reply:
We should

Reply:
> We should

Will it be done in this PR or next one in future?

Reply:
I will update the PR. Sorry for the confusion.